feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/README.md

```diff
@@ -22,117 +23,132 @@ Ensure that when **all LAG member** become set with forwarding-viable == FALSE.
 |            |            |             + - - - + + - - - -|              |
 |            |            |             + - - - + + - - - -|     .-------.|
 |.-------.   |            |             +-------+-+--------+    (  pfx2   )
-(  pfx1   )  |     .      |             | p7    : ;   p7   |     `-------'|
+(  pfx1   )  |     .      |             | p6    : ;   p6   |     `-------'|
 |`-------'   | p1 ; : p1  |     DUT     |        '         |     .-------.|
 |            |----+-+-----|             |                  |    (  pfx3   )
-|            |    | |     |             | p8     .    p8   |     `-------'|
+|            |    | |     |             | p7     .    p7   |     `-------'|
 |            |    | |     |             +-------;-:--------+     .-------.|
 |            |    : ;     |             |       | |        |    (  pfx4   )
 |            |     '      |             |       | |        |     `-------'|
 |            |  LAG_1     |             +-------+-+--------+              |
-+------------+            +-------------+ p9    : ;   p9   +--------------+
++------------+            +-------------+ p8    : ;   p8   +--------------+
                                                  '
                                                 LAG_3
 
 ```
 
-- Connect ATE port-1 to DUT port-1, and ATE ports 2 through 7 to DUT ports 2-7,
-  and ATE ports 8, 9 to DUT ports 8, 9
+- Connect ATE port-1 to DUT port-1, and ATE ports 2 through 8 to DUT ports 2-8
 - Configure ATE and DUT ports 1 to be LAG_1 w/ LACP running.
-- Configure ATE and DUT ports 2-7 to be LAG_2 w/ LACP running.
-- Configure ATE and DUT ports 8-9 to be LAG_3 w/ LACP running.
+- Configure ATE and DUT ports 2-6 to be LAG_2 w/ LACP running.
+- Configure ATE and DUT ports 7-8 to be LAG_3 w/ LACP running.
 - Establish ISIS adjacencies on LAG_1, LAG_2, LAG_3.
   1. Advertise one network prefix (pfx1) from ATE LAG_1
   1. Advertise one network prefix (pfx2) from ATE LAG_2 and ATE LAG_3.
-- Establish iBGP between ATE and DUT over LGA_1 using LAG_1 interface IPs and advertise prefix pfx3 with BGP NH from pfx2 range.
+- Establish iBGP between ATE and DUT over LGA using LAG interface IPs and 
+  advertise prefix pfx3 with BGP NH from pfx2 range.
 - Programm via gRIBI route for prefix pfx4 with single NHG pointing LAG_2 (al
```

Comment:
@dplore , i think the idea of forwarding-viable is switch to backup NHG if available when all member of primary are set forwarding-viable=False . I don't think we have that coverage.  Can we add backup NHG pointing to LAG3 and verify that gRIBI traffic switched to LAG3 when all member of LAG2 are set to False  and similarly it is switched back to primary ( LAG2) when any of the member set to forwarding-viable True .  ( PS: we can ignore this case here  if it planned as part of gRIBI  test cases)

Reply:
Please let me know what can be added so that i will change it accordingly.

Reply:
@rahulsy26 , Iam talking about gRIBI route traffic . that is getting dropped when all members of LAG2 are down.

Reply:
Lets wait for @dplore to comment on this

Reply:
Yes, we should include a backup-NHG.  We will have the scenario where Backup-NHG is present and a LAG will have all member interfaces set to forwarding-viable=FALSE.  The backup-NHG should take effect in this scenario.  @nflath for any other comment

Reply:
Noted

Reply:
Added NHG As per Nachikethas Suggested